src/browser/api_protocol/api_handlers/grouped_window_moves.ts

```diff
@@ -40,7 +40,7 @@ export function hijackMovesForGroupedWindows(actions: ActionSpecMap) {
                     const { payload } = message;
                     const { uuid, name } = getTargetWindowIdentity(payload);
                     const wrapped = getWindowByUuidName(uuid, name);
-                    if (wrapped && wrapped.groupUuid) {
+                    if (wrapped && wrapped.groupUuid && !payload.options.moveIndependently) {
```

Comment:
Check if options is undefined for old runtimes  adapters

Reply:
Done!

Reply:
:neckbeard: assign that to a variable rather than modifying payload